unfetter-discover/unfetter-store
# Error msg

Show better error message on bad csv file uploads

## To Test
* visit https://localhost:9443/#/threat-dashboard/create
* select import csv
* select any file
* verify an error message asking for certain
